network/encoding.go

```diff
@@ -36,7 +36,8 @@ var ErrorType = PacketTypeID(uuid.Nil)
 
 // String returns the canonical string representation of the PacketTypeID
 func (mId PacketTypeID) String() string {
-	return uuid.UUID(mId).String()
+	t, _ := registry.get(mId)
```

Comment:
It was for testing, so here we should be more careful, namely check the `ok` variable to see if it is present or not. If it's not present, we should just print the hexadecimal version then.